test/jdk/java/awt/dnd/InterJVMGetDropSuccessTest/InterJVMGetDropSuccessTest.java

```diff
@@ -226,6 +226,9 @@ public void dragDropEnd(DragSourceDropEvent dsde) {
         }
     }
 
+    private boolean success1 = false;
+    private boolean success2 = false;
```

Comment:
These boolean vars need to be volatile since they are accessed on two different threads.